docs/team/limjunxian1.md

```diff
@@ -5,42 +5,62 @@
+  * Added labels to each of the attributes in the `PersonCard` so that users are able to distinguish what the values are referring to.
+
+* **Bug fixing**: Updated specificity of error messages displayed for commands that has index as a parameter and disallowing `tag` to have duplicate values. (Pull request [#208](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/208))
+  
 ## Contributions to the User Guide
-
-to be added soon
+  * Added introduction. (Pull request [#22](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/22))
+  * Added documentation for the `add`, `addalt`, `delete` and `list` commands. (Pull request [#140](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/140))
+  * Updated planned enhancements section and front matters (quick start and start of features section). (Pull request [#140](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/140))
+  * Updated command summary. (Pull requests [#140](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/140), [#156](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/156))
+  * Cleaned up errors before PED (Pull requests [#156](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/156), [#158](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/158))
+  * Improved based on suggestion from PED, standardization of Table of Contents to have parallel language and summarized visual components as per CS2101 tutor feedback (Pull request [#209](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/209))
 
 ## Contributions to the Developer Guide
-
-to be added soon
+  * Updated NFRs and User Stories (Pull request [#33](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/33))
+  * Added implementation details for `addalt` and `edit` command. (Pull request [#105](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/105))
+  * Added prefix summary (Pull request [#105](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/105))
+  * Updated Use Cases (Pull requests [#33](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/33), [#143](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/143))
+  * Updated Person Model (Pull request [#143](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/143))
 
 ## Contributions to Team-Based Tasks
-
-to be added soon
+  * Added introduction in UserGuide.md (Pull request [#22](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/22))
+  * Updated NFRs and User Stories in DeveloperGuide.md (Pull request [#33](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/33))
+  * Update structure of README.md for better clarity (Pull request [#105](https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/105))
+  * Managed release [v1.3.trial](https://github.com/AY2324S1-CS2103T-T13-2/tp/releases/tag/v1.3.trial)
 
 ## Review / Mentoring Contributions
-
-to be added soon
+  * Pull Requests reviewed: 23 [(GitHub)](https://github.com/AY2324S1-CS2103T-T13-2/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3A%40me)
```

Comment:
```suggestion
  * Pull Requests reviewed: 23 [(GitHub)](https://github.com/AY2324S1-CS2103T-T13-2/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Alimjunxian1)
```
Derp, minus 5% contribution for this error

Reply:
thanks for pointing this out! fixed in 2e485eff